libantworld/route_ardin.cc

```diff
@@ -191,6 +192,21 @@ bool RouteArdin::load(const std::string &filename, bool realign)
         }
     }
 
+    // Initialise bounds to limits of underlying data types
+    std::fill_n(&m_MinBound[0], 2, meter_t(std::numeric_limits<meter_t::underlying_type>::max()));
+    std::fill_n(&m_MaxBound[0], 2, meter_t(std::numeric_limits<meter_t::underlying_type>::min()));
```

Comment:
I think you might be able to just write:
```
std::fill_n(&m_MinBound[0], 2, std::numeric_limits<meter_t>::max());
```
etc. The units lib has overloads for some numeric_limits functions